js/home.js

```diff
@@ -51,6 +52,8 @@ inputField.addEventListener("keypress", (e) => {
         searchTerm = "Palestine";
       fetchWeatherData(searchTerm);
     }
+    inputField.value=""
+    suggBox.textContent=""
```

Comment:
Add spaces between the assign symbol and other stuff 😅
`inputField.value = ""`
`suggBox.textContent = ""`